ubuntu_image/image.py

```diff
@@ -179,8 +176,7 @@ def partition(self, partnum, **sfdisk_args):
         # - check status of the returned CompletedProcess
         # - handle errors
         # - log stdout/stderr
-        run(args, input=input, stdout=PIPE, stderr=PIPE,
-            universal_newlines=True)
+        run(SPACE.join(args))
```

Comment:
and this broke MBR-based images, because sfdisk hangs forever waiting for input...